Updates changelog.md for 2.15.2 (#2527)
AzureAD/microsoft-identity-web: changelog.md

2.15.2
=========
- For the .NET 8 rc2 target framework, the IdentityModel dependencies have been updated to Identity.Model.*.7.0.3.

### Bug Fixes

> - Fixes a regression introduced in 2.15.0 where ID.Web was not waiting for all OnTokenValidated delegates to be executed. See issue[#2513](https://github.com/AzureAD/microsoft-identity-web/issues/2513).

Reviewer: It's not IdWeb that didn't wait (it's ASP.NET Core). But this was a regression because we were not chaining the events any longer.

```suggestion
- Fixes a regression introduced in 2.15.0 where the OnTokenValidated delegates were no longer chained with an await. See issue[#2513](https://github.com/AzureAD/microsoft-identity-web/issues/2513).
```
